apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+  alt: "A bottom sheet containing a heading and a list of action items.",
+  width: "360px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use to display semi-critical information.",
+            "Use where the user needs to perform one specific task.",
+            "Use them as contextual interactions.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for multi-step flows. Use a modal view instead."
+        ]
+    }
+} %}
+
+---
+
+## Modal
+
+A modal is a surface that overlays the page’s main content and is used to display information, gather input or confirm actions.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/modal/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/modal.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/modal-narrow.svg",
+  alt: "A modal containing a heading, multiple lines of text ant two buttons placed below the modal heading.",
+  width: "600px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them to display important information that needs user’s action or dismissal.",
+            "Use them when the user needs to perform one specific task.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for multi-step flows."
+        ]
+    }
+} %}
+
+---
+
+## Full page modal
+
+Also known as Modal View (iOS) or Scaffold View (Android) are a surface that overlays the page’s main content entirely and is used to display information, gather input or confirm actions.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/modal/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/frame.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/frame-narrow.svg",
+  alt: "A surface that overlays the page’s main content entirely and displays a list of payment methods.",
+  width: "360px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them for container multi-step interactions.",
+            "Use them when the user needs to perform various tasks.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them purely to display information. <br/> Use a bottom sheet instead.",
+            "Don’t use them for a contextual interaction. <br/> Use a bottom sheet instead."
+        ]
+    }
+} %}
+
+---
+
+## Notification
```

Comment:
Can we either alphabetically sort these components (so bottom sheet first) or group them? I.e. all types of notification together, all types of modal together

Reply:
Good point! 
Components have been updated and sorted alphabetically.